src/__tests__/screen.js

```diff
@@ -21,6 +21,49 @@ test('exposes queries that are attached to document.body', async () => {
   expect(screen.queryByText(/hello world/i)).not.toBeNull()
 })
 
+test('logs Playground URL that are attached to document.body', () => {
+  renderIntoDocument(`<div>hello world</div>`)
+  screen.logTestingPlaygroundURL()
+  expect(console.log).toHaveBeenCalledTimes(1)
+  expect(console.log.mock.calls[0][0]).toMatchInlineSnapshot(`
+    "Open this URL in your browser
+
+    https://testing-playground.com/#markup=DwEwlgbgfAFgpgGwQewAQHdkCcEmAenGiA"
+  `)
+})
+
+test('logs messsage when element is empty', () => {
+  screen.logTestingPlaygroundURL(render('').container)
```

Comment:
No need to render here:

```suggestion
  screen.logTestingPlaygroundURL(document.createElement('div'))
```